src/components/App/__x_tests__/smoke.js

```diff
@@ -0,0 +1,17 @@
+// Cannot get this test to pass. Hence the renamed __tests__ directory.
+// There is a problem with C3; see https://github.com/c3js/c3/issues/2129
+// but the recommended solutions do not work. Preserving this attempt in the
+// hope that some future effort can actually make it work.
+
```

Comment:
There are several tests I've previously been unable to get to work, [like this one](https://github.com/pacificclimate/climate-explorer-frontend/blob/22c63d4f2ad220ab83dd5336e67e12bf835b8a22/src/core/__tests__/util-test.js#L192). I like the **xit** function for this because sometimes tests are a nice bit of documentation, so it can be useful to leave them findable in the normal location.

Reply:
I wanted to use `xit` too, but the error occurs before the test -- during the phase when the component under test is being imported. Hence this.

Reply:
That makes sense.